packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -1,125 +1,108 @@
+        logger.debug('Deploying operator contract')
+        const operatorContract = await deployOperatorContract(chainConfig, operatorWallet, { operatorSharePercent: 10 })
+        logger.debug(`Operator deployed at ${operatorContract.address}`)
+        const operatorConfig = {
+            operatorContractAddress: toEthereumAddress(operatorContract.address),
+            provider,
+            theGraphUrl,
+            signer: operatorWallet,
+            maxSponsorshipsCount: 20,
+            minSponsorshipEarnings: 1 // full tokens
         }
-        return totalDiff
+        return { operatorWallet, operatorContract, operatorConfig }
     }
 
     beforeAll(async () => {
-        const client = createClient(STREAM_CREATION_KEY)
-        streamId1 = (await createTestStream(client, module)).id
-        streamId2 = (await createTestStream(client, module)).id
-        await client.destroy()
-    })
-
-    beforeEach(async () => {
-        provider = getProvider()
+        provider = new JsonRpcProvider(`http://${STREAMR_DOCKER_DEV_HOST}:8547`)
         logger.debug('Connected to: ', await provider.getNetwork())
 
-        token = new Contract(chainConfig.contracts.DATA, tokenABI) as unknown as TestToken
-
-        ({ operatorWallet, operatorContract } = await setupOperatorContract({
-            provider,
-            chainConfig,
-            theGraphUrl
-        }))
-
-        await (
-            await token.connect(operatorWallet).transferAndCall(operatorContract.address, parseEther(`${STAKE_AMOUNT * 2}`), operatorWallet.address)
+        const adminWallet = new Wallet(STREAM_CREATION_KEY, provider)
+        const streamRegistry = new Contract(chainConfig.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        
+        logger.debug('Creating stream for the test')
+        const createStreamReceipt = await (await streamRegistry.createStream(
+            `/operatorvaluewatchertest-${Date.now()}`,
+            '{"partitions":1}')
         ).wait()
-        for (const streamId of [streamId1, streamId2]) {
-            const sponsorship = await deploySponsorship(chainConfig, operatorWallet, { streamId })
-            await (await token.connect(operatorWallet).transferAndCall(sponsorship.address, parseEther(`${SPONSOR_AMOUNT}`), '0x')).wait()
-            await (await operatorContract.stake(sponsorship.address, parseEther(`${STAKE_AMOUNT}`))).wait()
-        }
+        streamId = createStreamReceipt.events?.find((e) => e.event === 'StreamCreated')?.args?.id
+        const streamExists = await streamRegistry.exists(streamId)
+        logger.debug('Stream created:', { streamId, streamExists })
+        // TODO: use createClient once configs allow it. For now I'm creating the stream directly using the contract
+        // const client = createClient(STREAM_CREATION_KEY)
+        // streamId = (await createTestStream(client, module)).id
+        // await client.destroy()
```

Comment:
fast-chain is enabled in client now in streamr-1.0 branch. So you can use `client` to create streams.

Reply:
Updated.